### PR TITLE
minor improvements to developer docs for Scaladoc tool

### PR DESCRIPTION
**Pull request by SethTisue**
this comes out of the notes I made while working on #15927 with @yzia2000 and @jan-pieter -- can either of you guys think of anything else we should add?

### CODE REVIEW

**Comment by yzia2000** [comment]
Thanks for doing this!

File: scaladoc/README.md [hunk 1]
```diff
@@ -63,10 +63,17 @@ You can also find the result of building the same sites for latest `master` at:
...
 
+You can run the Scaladoc tool from the sbt prompt as a user would run it
+with `scaladoc/run`, passing `-d <directory>` and some `.tasty` files as
+input on the command line.
+
 ### Testing
 
-Most tests rely on comparing signatures (of classes, methods, objects etc.) extracted from the generated documentation
-to signatures found in source files. Such tests are defined using [SignatureTest](test/dotty/tools/scaladoc/signatures/SignatureTest.scala) class
+From the sbt prompt, `scaladoc/test` runs the tests.
```
Comment (yzia2000) [inline scaladoc/README.md:72]: Do we also want to include that we can run specific signature tests by adding the path as the argument or is that too verbose?

```
scaladoc/test *scaladoc.signatures.MarkdownCode
```

File: scaladoc/README.md [hunk 2]
```diff
@@ -63,10 +63,17 @@ You can also find the result of building the same sites for latest `master` at:
...
+
+Many tests simply consist of small Scala source files to be documented. In this style of test, it's not necessary for expected output to be included, because the test is its own specification. The testing infrastructure compares the signatures (of classes, methods, objects etc.) extracted from the generated documentation to the signatures extracted from the same source file using Scalameta.
+
+Such tests are defined using [SignatureTest](test/dotty/tools/scaladoc/signatures/SignatureTest.scala) class
 and its subtypes (such as [TranslatableSignaturesTestCases](test/dotty/tools/scaladoc/signatures/TranslatableSignaturesTestCases.scala))
```
Comment (yzia2000) [inline scaladoc/README.md:77]: I am not sure if its useful but maybe you can include that the source files are actually present in the `scaladoc-testcases` subproject in `src/tests`

**Comment by Sporarum** [comment]
Amazing work !

In the same spirit, I would encourage you to add the command to https://docs.scala-lang.org/scala3/guides/contribution/procedures-testing.html

**Comment by ckipp01** [comment]
I don't want perfect to get in the way of better. @SethTisue is there anything else you wanted to add to this, or is it good to be reviewed?

**Comment by SethTisue** [comment]
@ckipp01 I'll try to get back to this within the next couple weeks

File: scaladoc/README.md (hunk 2 above)
Comment (SethTisue) [reply scaladoc/README.md]: done

File: scaladoc/README.md (hunk 1 above)
Comment (SethTisue) [reply scaladoc/README.md]: added

**Comment by SethTisue** [comment]
> I would encourage you to add the command to [docs.scala-lang.org/scala3/guides/contribution/procedures-testing.html](https://docs.scala-lang.org/scala3/guides/contribution/procedures-testing.html)

Good suggestion — I added a link from that page to this page.

**Review by ckipp01 — APPROVED** [review]
Thanks for this @SethTisue. LGTM!

**Pull request merged**